.github/workflows/push.yaml

```
@@ -91,6 +91,14 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           cache: false
 
+      - name: install operator-sdk
+        run: |
+          export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+          export OS=$(uname | awk '{print tolower($0)}')
```

You can pull these straight from `GOENV` and `GOARCH`.
```suggestion
          export ARCH="$(go env GOARCH)"
          export OS="$(go env GOOS)"
```